en/docs/consuming-services/consuming-a-sevice.md

```diff
@@ -0,0 +1,94 @@
+# Consume a Service
+
+Choreo is a platform that allows you to create, deploy, and consume services seamlessly. The Choreo developer portal simplifies the process of discovering and using APIs for API consumers. 
+
+Typically, an application developer who may be internal or external to your organization would want to consume the APIs published in the developer portal to develop their applications. In this guide, you will learn how to discover, generate credentials, and test the consumption of a service published in the Choreo Developer Portal. You will also learn how to invoke the service via a web application. 
+
+This guide walks you through the following steps:
+
+- Create an application in the Developer Portal and generate credentials for the application. 
+- Subscribe the API to the application.
+- Consume a published REST API via your web application.
+
+## Prerequisites
+
+Before you try out this guide, if you do not have a published API/ service that you would want to consume via the developer portal, follow [Develop a REST API ](../develop-components/develop-services/develop-a-rest-api.md) documentation to publish and deploy a sample REST API. 
+
+## Discover APIs
+
+In the Choreo Developer Portal, developers can use the search option to find APIs by it's name. The APIs and services that are created and published through the Choreo console, become visible in the Developer Portal depending on the APIs visibility as follows:
```

Comment:
Need to change 
- `by it's name` to `by its name` or it'll be ok to just say `by name`.
- `Choreo console` to `Choreo Console`. We also don't seem to need the comma before `become` right?